components/rewards/RewardsHome.js

```diff
@@ -25,11 +27,31 @@ function createList(n) {
   return list;
 }
 
-function RewardsHome({ customer }) {
+function RewardsHome({ customer, participating }) {
+  const navigation = useNavigation();
   const rewardsAvailable = parseInt(customer.points, 10) / rewardPointValue;
   const pointsToNext = parseInt(customer.points, 10) % rewardPointValue;
   return (
     <ScrollView style={{ marginLeft: 16, paddingRight: 16 }}>
+      <ColumnContainer>
+        <Overline style={{ marginTop: 24, marginBottom: 4 }}>
+          Participating Stores
+        </Overline>
+        {participating.map((store) => {
+          return (
+            <Subhead
+              style={{ marginLeft: 8 }}
+              key={store.id}
+              onPress={() =>
+                navigation.navigate('Stores', {
```

Comment:
Hmm this is really really slow to load for some reason. Could we do first do a `navigation.goBack` then do `changeCurrentStore` from MapScreen to set the store if that's not too clunky?

Reply:
Just so it doesn't leave the user staring at the screen wondering what's happening/what's supposed to happen.

Also, can you wrap these in touchable opacities so there's some response when these are clicked? Even that might help indicate that some action is supposed to happen bc rn it's quite ambiguous.

Reply:
> Could we do first do a navigation.goBack then do changeCurrentStore from MapScreen to set the store if that's not too clunky?

AFAIK this isn't possible? The method that gets invoked is `componentWillReceiveProps`, which is setting the store in `MapScreen`

> can you wrap these in touchable opacities so there's some response when these are clicked

👍